.github/workflows/docs.yml

```
@@ -52,21 +48,21 @@ jobs:
         run: yarn build
 
       - name: Generate the sitemap
-        uses: cicirello/generate-sitemap@v1
+        uses: cicirello/generate-sitemap@f76c8312a5a364c49e2610b8af12ae861805a056
         with:
           path-to-root: './docs/build'
           drop-html-extension: true
-          base-url-path: https://onboard.blocknative.com
+          base-url-path: https://web3onboard.thirdweb.com
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)